### ISSUE TITLE
Add ability to specify suppression options on SES configuration set

### ISSUE THREAD

**Issue by JoshiiSinfield** [hashicorp/terraform-provider-aws#14320]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

We'd like to change the default configuration for destination suppression config on an SES Configuration set. 

This can also be done at an account level. I see that as being separate, and feel that having blocks within the `aws_ses_configuration_set` resource makes more sense. Then having a separate resource for account-level settings.

This request is purely for config sets.

### New or Affected Resource(s)

* aws_ses_configuration_set

### Potential Terraform Configuration

```hcl
resource "aws_ses_configuration_set" "only_suppress_bounce" {
  name = "my-config-set"
  
  suppression_options {
    suppressed_reasons = ["BOUNCE"]
  }
}
```

### References

* https://docs.aws.amazon.com/sdk-for-go/api/service/sesv2/#SESV2.PutConfigurationSetSuppressionOptions
* or v2: https://docs.aws.amazon.com/sdk-for-go/v2/api/service/sesv2/#Client.PutConfigurationSetSuppressionOptionsRequest

If I can find some time I'll push a PR.

**Comment by DrFaust92** [comment]
This seems to be only supported for SESv2

**Comment by kamilturek** [comment]
Resolved by #27056.
